src/test/java/homeworkCSV/DataTest.java

```diff
@@ -0,0 +1,56 @@
+package homeworkCSV;
+
+import java.util.LinkedList;
+
+import static org.junit.Assert.assertEquals;
+
+
+class DataTest {
+
+
+    @org.junit.jupiter.api.Test
+    void getDataFromCSV() {
+    }
+
+    @org.junit.jupiter.api.Test
```

Comment:
create tests for adding 0 runners, for adding 2 runners, for adding 2 runners and check sorting by total time